Include ppd.h so that ppd_file_t is properly defined at build time.

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>

--- hplip-3.16.7/prnt/hpcups/HPCupsFilter.h.orig	2016-08-26 16:15:49.626710267 +0000
+++ hplip-3.16.7/prnt/hpcups/HPCupsFilter.h	2016-08-26 16:16:29.646709482 +0000
@@ -31,6 +31,7 @@
 #ifndef HP_CUPSFILTER_H
 #define HP_CUPSFILTER_H
 
+#include <cups/ppd.h>
 #include "CommonDefinitions.h"
 #include "SystemServices.h"
 #include "Compressor.h"
--- hplip-3.16.7/prnt/hpijs/hpcupsfax.cpp.orig	2016-08-26 16:20:21.870704928 +0000
+++ hplip-3.16.7/prnt/hpijs/hpcupsfax.cpp	2016-08-26 16:20:42.482704524 +0000
@@ -43,6 +43,7 @@
 #include <arpa/inet.h>
 #include <math.h>
 #include <cups/cups.h>
+#include <cups/ppd.h>
 #include <cups/raster.h>
 #include <string>
 #ifdef FALSE
